projects/manageiq.py

```diff
@@ -96,3 +98,23 @@ def start():
 def stop():
     with cd('manageiq/vmdb'):
         run('bundle exec rake evm:kill')
+
+
+@task
+def create_provider(name, hostname, credentials):
+    body = {
+        "action": "create",
+        "resources": [
+            {
+                'name': name,
+                'type': 'EmsKubernetes',
+                'hostname': hostname,
+                'port': '8443'
+            }]
+        }
+    data = json.dumps(body)
```

Comment:
Can be simplified with:

    data = json.dumps({
        "action": "create",
        "resources": [
            {
                'name': name,
                'type': 'EmsKubernetes',
                'hostname': hostname,
                'port': '8443'
            }]
        })